crates/sui-benchmark/tests/simtest.rs

```diff
@@ -64,7 +64,12 @@ mod test {
     async fn test_simulated_load_basic() {
         sui_protocol_config::ProtocolConfig::poison_get_for_min_version();
         let test_cluster = build_test_cluster(7, 0).await;
-        test_simulated_load(test_cluster, 15).await;
+        tokio::time::timeout(
+            Duration::from_secs(30),
+            test_simulated_load(test_cluster, 15),
```

Comment:
What's this change for?

Reply:
I had a bug earlier where this call would run forever. Adding a timeout just so that it's easier to debug similar issues.

Reply:
can you move the timeout inside of `test_simulated_load`, so that it respects the duration passed in (instead of a hard coded timeout), and covers all tests?